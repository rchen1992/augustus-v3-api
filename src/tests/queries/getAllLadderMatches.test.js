const mockLadderRepo = require('@tests/mockRepos/ladder');
const mockMatchRepo = require('@tests/mockRepos/match');
const { testQuery } = require('@tests/utils');

testQuery({
    id: "Get all of a ladder's matches",
    query: `
        {
            ladder(id: 1) {
                id
                matches {
                    id
                    createdAt
                }
                matchCount
            }
        }
    `,
    variables: {},
    context: () => {
        return {
            repos: {
                ladder: mockLadderRepo,
                match: mockMatchRepo,
            },
        };
    },
    expected: undefined,
});
