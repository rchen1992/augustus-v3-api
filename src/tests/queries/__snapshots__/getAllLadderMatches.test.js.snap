// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`query: Get all of a ladder's matches 1`] = `
Object {
  "data": Object {
    "ladder": Object {
      "id": "1",
      "matchCount": 3,
      "matches": Array [
        Object {
          "createdAt": "2018-12-07 05:46:42",
          "id": "1",
        },
        Object {
          "createdAt": "2018-12-06 05:46:42",
          "id": "6",
        },
        Object {
          "createdAt": "2018-11-03 05:46:42",
          "id": "7",
        },
      ],
    },
  },
}
`;
